pkg/storaged/stratis-panel.jsx

```diff
@@ -119,10 +120,24 @@ export function create_stratis_pool(client) {
                           value: name,
                           validate: name => validate_pool_name(client, null, name)
                       }),
-            CheckBoxes("encrypt", "",
+            SelectSpaces("disks", _("Block devices"),
+                         {
+                             empty_warning: _("No block devices are available."),
+                             validate: function (disks) {
+                                 if (disks.length === 0)
+                                     return _("At least one block device is needed.");
+                             },
+                             spaces: get_available_spaces(client)
+                         }),
+            CheckBoxes("encrypt_pass", client.features.stratis_crypto_binding ? "Encryption" : "",
                        {
                            fields: [
-                               { tag: "on", title: _("Encrypt data") }
+                               {
+                                   tag: "on",
+                                   title: (client.features.stratis_crypto_binding
+                                       ? _("Use a passphrase")
+                                       : _("Encrypt data"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230608-133956-15033e2f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)